### DISCUSSION TITLE
CREATE PROCEDURE NOT WORKING IN SQL

### DISCUSSION THREAD

**Discussion by Kola-Ademola** [sidorares/node-mysql2#1537] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/60304620/159117943-de0cc5bd-b796-4e4d-b0f1-4bb3eb2461f3.jpeg)

Hi everyone have been trying to create a procedure but I keep getting a syntax error any ideas how I can solve this..? All help is appreciated thanks..!!!!

**Comment by naveen1337** [comment, marked as answer]
Is that missing semicolon at line 75 where the end SQL statement causes ? 

also the line 77 the for resetting the delimiter back to normal, the space is missing between the keyword 

[https://www.mysqltutorial.org/getting-started-with-mysql-stored-procedures.aspx](https://www.mysqltutorial.org/getting-started-with-mysql-stored-procedures.aspx)

**Reply by Kola-Ademola** [reply]
Wow  thank yhu so much that helped lol have been looking at it for hours but didnt spot the missing semi colon lmao
